controllers/osbuild_controller.go

```diff
@@ -18,19 +20,51 @@ package controllers
 
 import (
 	"context"
+	"fmt"
+	"github.com/go-logr/logr"
+	"net/http"
+	"time"
 
+	"k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/runtime"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/log"
 
-	osbuilderprojectflottaiov1alpha1 "github.com/project-flotta/osbuild-operator/api/v1alpha1"
+	osbuildv1alpha1 "github.com/project-flotta/osbuild-operator/api/v1alpha1"
+	"github.com/project-flotta/osbuild-operator/internal/composer"
+	repositoryosbuild "github.com/project-flotta/osbuild-operator/internal/repository/osbuild"
+)
+
+const (
+	// Conditions Messages
+	FailedToSendPostRequestMsg      = "Failed to post a new composer build request"
```

Comment:
Should all these consts be exposed?

Reply:
In the next phase the controller will send a postRequest with edge-installer
I guess I'll use the same messages

Reply:
But, it will still be in the same controller, no?